javascript/net/grpc/web/grpcwebclientbase.js

```diff
@@ -123,18 +123,18 @@ class GrpcWebClientBase {
       let unaryStatus;
       let unaryMsg;
       GrpcWebClientBase.setCallback_(
-          stream, (error, response, status, metadata) => {
+          stream, (error, response, status, metadata, trigger) => {
```

Comment:
could you rename `trigger` to a more explicit name?

Maybe `unaryEndOfStream`?

Reply:
renamed.